app/models/inventory_improvement/project.rb

```diff
@@ -0,0 +1,12 @@
+module InventoryImprovement
+  class Project < ApplicationRecord
+    include ::GenericToCsv
```

Comment:
Shouldn't we just include this in the ApplicationRecord?

Reply:
I think it's unlikely all models are going to need it.